<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VIIGL0A" failures="2" tests="12" name="Test" time="0.11" errors="0" timestamp="04 Aug 2021 03:01:59 GMT">
  <testcase classname="Test.RetryTest" name="test1" time="0.021">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="Test.RetryTest" name="test1" time="0.001">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="Test.RetryTest" name="test1" time="0.002">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="Test.RetryTest" name="test1" time="0.001">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="Test.RetryTest" name="test1" time="0.002">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at Test.RetryTest.test1(RetryTest.java:10)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="Test.RetryTest" name="test2" time="0.002"/>
  <testcase classname="Test.RetryTest" name="test3" time="0.001">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="Test.RetryTest" name="test3" time="0.001">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="Test.RetryTest" name="test3" time="0.002">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="Test.RetryTest" name="test3" time="0.001">
    <skipped/>
  </testcase> <!-- test3 -->
  <testcase classname="Test.RetryTest" name="test3" time="0.002">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at Test.RetryTest.test3(RetryTest.java:20)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="Test.RetryTest" name="test4" time="0.002"/>
</testsuite> <!-- Test -->
